pkgs/top-level/all-packages.nix

```diff
@@ -19919,7 +19919,10 @@ with pkgs;
 
   rizinPlugins = recurseIntoAttrs rizin.plugins;
 
-  cutter = libsForQt5.callPackage ../development/tools/analysis/rizin/cutter.nix { };
+  cutter = libsForQt5.callPackage ../development/tools/analysis/rizin/cutter.nix {
+    ## as of 2023-10-27: pyside2 is unsupported on python 3.11
```

Comment:
Pyside2 does actually officially support Python 3.11 as of the 5.15.11 release last month. The cutter build still seems to fail, however, because pyside2 has a hardcoded max version in one of the headers, but only when `Py_LIMITED_API` is set (and cutter seems to be the only dependent to set that)

Do you think you could maybe make this comment clearer that it seems to be a cutter-specific error, since all other dependent packages now seem to build fine with #263599 (https://github.com/NixOS/nixpkgs/pull/263599#issuecomment-1793354820)?

Thank you!